dxr/plugins/rust/tests/test_data/test_fields.py

```diff
@@ -52,29 +52,29 @@ def test_qual_enum_field_ref(self):
         self.found_line_eq('+var-ref:test::EnumFields::Struct::field1', "EnumFields::Struct{ <b>field1</b>, field2 } =&gt; {}", 42)
         self.found_line_eq('+var-ref:test::EnumFields::Struct::field2', "EnumFields::Struct{ field1, <b>field2</b> } =&gt; {}", 42)
 
-    def test_field_ref(self):
```

Comment:
(Per IRC) changes to rustc broke stuff, and we haven't noticed because our copy is pinned to a June release.